README.md

```diff
@@ -31,57 +30,56 @@ store = new Store(dbName, options)
-api.findAll()
-api.findAll(filterFunction)
-api.update(id, changedProperties)
-api.update(id, updateFunction)
-api.update(object)
-api.update([object1, id2])
-api.updateOrAdd(id, object)
-api.updateOrAdd(object)
-api.updateOrAdd([object1, id2])
-api.updateAll(changedProperties)
-api.updateAll(updateFunction)
-api.remove(id)
-api.remove(object)
-api.remove([object1, id2])
-api.removeAll()
-api.removeAll(filterFunction)
-api.clear() // removes all data without triggering events
+store.add(object)
+store.add([object1, id2])
+store.find(id)
+store.find(object) // with id property
+store.findOrAdd(id, object)
+store.findOrAdd(object)
+store.findOrAdd([object1, id2])
+store.findAll()
+store.findAll(filterFunction)
+store.update(id, changedProperties)
+store.update(id, updateFunction)
+store.update(object)
+store.update([object1, id2])
+store.updateOrAdd(id, object)
+store.updateOrAdd(object)
+store.updateOrAdd([object1, id2])
+store.updateAll(changedProperties)
+store.updateAll(updateFunction)
+store.remove(id)
+store.remove(object)
+store.remove([object1, id2])
+store.removeAll()
+store.removeAll(filterFunction)
+store.clear() // removes all data without triggering events
 
 // sync methods, return native promises
-hoodie.store.pull() // pulls changes one-time
-hoodie.store.push() // pushes changes one-time
-hoodie.store.sync() // pulls and pushes changes one-time
-hoodie.store.connect() // starts continuous replication
-hoodie.store.disconnect() // stops continuous replication and all pending requests
+store.pull() // pulls changes one-time
+store.push() // pushes changes one-time
+store.sync() // pulls and pushes changes one-time
+store.connect() // starts continuous replication
+store.disconnect() // stops continuous replication and all pending requests
 // pull / push / sync can be filtered by id, object, array
-hoodie.store.pull('objectId')
-hoodie.store.push(object)
-hoodie.store.push(['object1Id', object2])
+store.pull('objectId')
+store.push(object)
+store.push(['object1Id', object2])
 
-// returns true or false if there are objects with unsynced changes
-hoodie.store.hasLocalChanges()
-// returns true or false if passed object (or id) has unsynced changes
-hoodie.store.hasLocalChanges(id)
-hoodie.store.hasLocalChanges(object)
+store.hasLocalChanges()
+.then(function (changedObjects) {
+
+})
 
 // events
-hoodie.store.on('add', function(object, options) {})
-hoodie.store.on('update', function(object, options) {})
-hoodie.store.on('remove', function(object, options) {})
-hoodie.store.on('change', function(eventName, object, options) {})
-hoodie.store.on('sync', function(object) {})
-hoodie.store.on('clear', function() {})
-hoodie.store.one(event, handler)
-hoodie.store.off(event, handler)
+store.on('add', function(object, options) {})
+store.on('update', function(object, options) {})
+store.on('remove', function(object, options) {})
+store.on('change', function(eventName, object, options) {})
+store.on('sync', function(object) {})
```

Comment:
instead of sync, we now have

- pull
- push

and also

- connect
- disconnect

see https://github.com/hoodiehq/pouchdb-hoodie-sync#usage